### DISCUSSION TITLE
ICSx⁵ 2.0 released

### DISCUSSION THREAD

**Discussion by rfc2822** [bitfireAT/icsx5#22] [Announcements]
**Version 2.0**

* support file access (local and remote storage) over Storage Access Framework (SAF) – does not even need storage permissions anymore (if you have old `file://` subscriptions, remove them and add them again over SAF)
* minor improvments and bug fixes

![2-0-announcement](https://user-images.githubusercontent.com/156167/156803885-d655c567-b7b3-4b93-a088-eedb779ecf3f.png)

**Comment by rfc2822** [comment]
**Version 2.0.1**

* hotfix for "Requires authentication" when adding a subscription
* supports color extraction from Webcal feeds
* minor improvements and bug fixes

**Reply by rfc2822** [reply]
**Version 2.0.2**

* exceptions of unknown recurring events are now correctly synchronized
* uses WorkManager for synchronization
* allows use of user-installed CAs

**Comment by devvv4ever** [comment]
**Version 2.0.3**

* update iCalendar processing (ical4android) - this also fixes problems when no color for the calendar was provided
* update other libraries
* use coroutines for HTTP processing
